climate_categories/data/conversion.FAO.IPCC2006_PRIMAP.csv

```diff
@@ -0,0 +1,20 @@
+# references: FAOSTAT https://files-faostat.fao.org/production/GT/Mapping_to_UNFCCC_IPCC.pdf
+# last_update: 2025-01-10
+FAO,gas,IPCC2006_PRIMAP,comment
+4,CO2,3.B.1,Carbon stock change in forests
+4.B,CO2,M.NFC,Net Forest conversion
+5.A,CO2 N2O,3.B.3,Drained grassland
```

Comment:
This is a subcategory of Grassland remaining Grassland 3.B.3.a, so I think it should be mapped to 3.B.3.a to make clear that Land converted to Grassland is not covered.